src/sentry/integrations/github/integration.py

```diff
@@ -252,11 +248,6 @@ def has_repo_access(self, repo: RpcRepository) -> bool:
             return False
         return True
 
-    def get_commit_context_all_frames(
```

Comment:
Removing these implementations from the github/gitlab integration files and putting it in the mixin since there should be no need to customize it per integration (it just calls get_blame_for_files)